src/DropdownToggle.js

```diff
@@ -31,11 +40,13 @@ export default class DropdownToggle extends React.Component {
 DropdownToggle.defaultProps = {
   open: false,
   useAnchor: false,
-  bsRole: 'toggle'
+  bsRole: 'toggle',
+  bsClass: 'dropdown'
```

Comment:
I've actually been going through and swapping everything out for `bsClass` on my "update for React v15.2.0" branch.